addons/website/controllers/main.py

```diff
@@ -224,6 +224,18 @@ def social(self, social, **kwargs):
         if not url:
             raise werkzeug.exceptions.NotFound()
         return request.redirect(url)
+    
+    @http.route('/website/get_suggested_link', type='json', auth="user", website=True)
+    def get_suggested_link(self, needle):
+        current_website = request.website
+        installed_controllers = current_website.get_suggested_controllers()
+        last_created_pages = current_website.get_website_pages(order='write_date', limit=5).mapped(lambda x: (x.url))
+        matching_pages = current_website.search_pages(needle, limit=15)
+        return{
+            'controllers': installed_controllers,
+            'lastCreatedPages': last_created_pages,
```

Comment:
a reason to don't return lastCreatesPages record instead of url ? similary to matchingPages